docs/snippets/getting-started/inventory.yml

```diff
@@ -0,0 +1,20 @@
+anta_inventory:
+  hosts:
+  - host: 192.168.0.10
+    name: s1-spine1
+    tags: ['fabric', 'spine']
```

Comment:
If we removed `--tags` from the commands should we remove them from here as well? We don't talk about tags anywhere in the Getting Started.

Reply:
not necessarily